GuessingGame.Tests/UnitTest1.cs

```diff
@@ -7,7 +7,7 @@ namespace GuessingGame.Tests;
 public class UnitTest1
 {
     [Fact]
-    public void GetResult_found()
+    public void GetResult_Win()

```

Comment:
Its also a good idea to test for the loss condition as well, that way you know you code is working correctly.

You might also need another test to make sure that given a specific difficulty it returns the correct minimum and maximum range, that way you can be sure your code also handles that important business case.

Reply:
I am not getting this comment please explain me.

Reply:
You have a unit test for the win condition but you should also have a unit test for the Loss condition too.

Your test will check whether the guess value matches the random number but there is another piece of important logic which involves using the correct min and maximum values for the difficulty chosen. ie If you pick easy then it should correctly pick the right maximum and minimum values, This test would make sure you didn't accidentally get 1-20 for picking easy. I wouldn't worry about this test as much, just something to think about when unit testing.